Gemfile.lock

```diff
@@ -254,7 +254,7 @@ GEM
     thread_safe (0.3.6)
     typhoeus (1.4.0)
       ethon (>= 0.9.0)
-    tzinfo (1.2.9)
```

Comment:
Left reviews and line comments
Created a file and made a commit
Opened a pull request